jkube-kit/resource/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceTest.java

```diff
@@ -46,37 +46,37 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-public class HelmServiceTest {
+class HelmServiceTest {
 
   private HelmConfig.HelmConfigBuilder helmConfig;
   private JKubeConfiguration jKubeConfiguration;
   private HelmService helmService;
 
-  @Before
-  public void setUp() throws Exception {
+  @BeforeEach
+  void setUp() {
     helmConfig = HelmConfig.builder();
     jKubeConfiguration = JKubeConfiguration.builder()
         .registryConfig(RegistryConfig.builder().settings(new ArrayList<>()).build()).build();
     helmService = new HelmService(jKubeConfiguration, new KitLogger.SilentLogger());
   }
 
-  @After
-  public void tearDown() {
+  @AfterEach
+  void tearDown() {
     helmService = null;
     helmConfig = null;
   }
 
-  @Test(expected = IOException.class)
-  public void prepareSourceDirValidWithNoYamls() throws Exception {
+  @Test
+  void prepareSourceDirValidWithNoYamls() {
     File file = mock(File.class);
     // Given
     when(file.isDirectory()).thenReturn(true);
     // When
-    HelmService.prepareSourceDir(helmConfig.build(), HelmConfig.HelmType.OPENSHIFT);
+    assertThrows(IOException.class, () -> HelmService.prepareSourceDir(helmConfig.build(), HelmConfig.HelmType.OPENSHIFT));
```

Comment:
It would be nice to add a little bit more validation about the exception .

And maybe provide a better test description for the tested cases:
- prepareSourceDir with nonexistent directory should throw exception
- prepareSourceDir with empty directory should throw exception
- prepareSourceDir with directory with no YAML files should throw exception

or things along those lines.